src/config/index.js

```diff
@@ -37,5 +37,4 @@ if (firebase.messaging.isSupported()) {
 }
 
 export const messaging = firebase_messaging;
-
```

Comment:
what is the change in this file?

Reply:
i guess i made a wrong pull i will make a new one

Reply:
I have updated the changes i think I missed it previously.